rclpy/executors/examples_rclpy_executors/callback_group.py

```diff
@@ -16,14 +16,18 @@
 import rclpy
 from rclpy.callback_groups import MutuallyExclusiveCallbackGroup
 from rclpy.executors import MultiThreadedExecutor
+import rclpy.logging
```

Comment:
Does it make sense to require users to import the logging module to use it, or should we always import it in rclpy's init instead?

Reply:
The user still needs to call a function from it to create a logger instance. So what gain would we get from an implicit import during init?

Reply:
that the diff for this PR would not include this line.

Without the implicit import, even users that just want to use the root logger have to import `rclpy.logging` to call `rclpy.logging.loginfo()` (unlike in rospy). It seems convenient to always have it imported but I am asking because it might not be good from a design perspective

Reply:
My personal opinion is "import what you use" but that is certainly different from what is done in `rospy`. I will leave it up to others to give feedback on this.

Reply:
I don't care one way or the other. I wouldn't be surprised/shocked to see that `import rclpy` allows me to do `rclpy.logging.*`, which is how `import os; os.path.join(...)` works. On the other hand, it would be fine to have to explicitly import logging too.

On balance, I'd slightly prefer not needing this import so the user's code is slightly shorter and I figure it will be a common enough thing to save a non-trivial amount of typing on user's part.